sessionspaces/src/resources/config_maps.rs

```diff
@@ -25,15 +25,26 @@ pub async fn create_configmap(
             session.proposal_number.to_string(),
         ),
         ("visit".to_string(), session.visit.to_string()),
-        ("beamline".to_string(), session.beamline),
+        ("beamline".to_string(), session.beamline.clone()),
         (
             "members".to_string(),
             serde_json::to_string(&session.members)?,
         ),
+        ("start_date".to_string(), session.start_date.to_string()),
+        ("end_date".to_string(), session.end_date.to_string()),
     ]);
     if let Some(gid) = session.gid {
         configmap_data.insert("gid".to_string(), gid);
     }
+    if BL_WITH_PATH.contains(&&session.beamline[..]) {
```

Comment:
Just had a play... this works and is (in my opinion) more readable:
```suggestion
    if BL_WITH_PATH.contains(&session.beamline.as_str()) {
```